tests/unit/commands/local/lib/test_local_api_service.py

```diff
@@ -181,39 +154,3 @@ def test_must_return_none_if_path_not_exists(self, os_mock):
 
         result = LocalApiService._make_static_dir_path(cwd, static_dir)
         self.assertIsNone(result)
-
-
-class TestRoutingList(TestCase):
```

Comment:
Similar here. Why was this removed? Do we still have this covered in other areas.

Reply:
Similar reasoning to the other comment